rebar.config

```diff
@@ -60,7 +60,7 @@
   {jsx, "1\.4\..*", {git, "git://github.com/talentdeficit/jsx", "v1.4.2"}},
 
   %% for testing
-  {proper, "1\.1", {git, "git://github.com/manopapad/proper", "v1.1"}},
+  {proper, "1\.1", {git, "git://github.com/manopapad/proper", {branch, "master"}}}, %"v1.1"}},
```

Comment:
branch master sounds dangerous. Maybe you could specify a commit?

I think there was some way to generate and use a rebar lock file, that would also be a possible solution.